ui/styles.css

```
@@ -212,3 +212,49 @@ body {
+  flex-direction: column;
+  gap: 2px;
+  padding: 12px;
+  background: var(--ebay-bg-alt);
+  border: 1px solid var(--ebay-border);
+  border-radius: var(--ebay-radius);
+}
+
+.stat-label {
+  color: var(--ebay-text-soft);
+  font-size: 12px;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)